components/movementsensor/gpsrtk/gpsrtkserial_test.go

```diff
@@ -74,31 +74,47 @@ func TestValidateRTK(t *testing.T) {
 	})
 }
 
-func TestReconfigure(t *testing.T) {
-	g := &rtkSerial{
-		writePath: "/dev/ttyUSB0",
-		wbaud:     9600,
-		logger:    logging.NewTestLogger(t),
+func TestValidatePMTKRTK(t *testing.T) {
+	path := "path"
+	validConfig := I2CConfig{
```

Comment:
The file containing this struct only builds in linux. Can you move these tests to run in linux only as well or move the configs to a common file and move their validation tests along with them

Reply:
oh, very good catch! Thanks